swirl/processors/mapping.py

```diff
@@ -137,8 +169,11 @@ def process(self):
                                                 else:
                                                     payload[swirl_key+"_"+source_key] = str(parser.parse(result_dict[source_key]))
                                             else:
-                                                if swirl_result[swirl_key] == "":
```

Comment:
Need to investigate this, bc if swirl_key isnt in swirl_result, this will explode in a shower of stack traces